webapp/channels/src/components/user_settings/general/user_settings_general.tsx

```diff
@@ -1276,6 +1336,124 @@ export class UserSettingsGeneralTab extends PureComponent<Props, State> {
         );
     };
 
+    createCustomAttributeSection = () => {
+        const attributeSections = this.props.customProfileAttributes.map((attribute) => {
+            let attributeValue = '';
+            if (attribute.id in this.state.customAttributeValues) {
+                attributeValue = this.state.customAttributeValues[attribute.id];
+            }
+            const sectionName = 'customAttribute_' + attribute.id;
+            const active = this.props.activeSection === sectionName;
+            let max = null;
+
+            if (active) {
+                const inputs = [];
+
+                let extraInfo: JSX.Element|string;
+                const isHandled = this.props.samlPositionAttributeSet;
+                if (isHandled === true) {
+                    extraInfo = (
+                        <span>
+                            <FormattedMessage
+                                id='user.settings.general.field_handled_externally'
+                                defaultMessage='This field is handled through your login provider. If you want to change it, you need to do so through your login provider.'
+                            />
+                        </span>
+                    );
+                } else {
```

Comment:
This looks unbound: if `samlPositionAttributeSet` is true, all custom fields will repeat this same message which seems out of place (except for a future sync'd `position` field?).

Also, when/if that block is valid, the save button should be hidden I think. Currently seeing:
![CleanShot 2025-01-02 at 16 33 23](https://github.com/user-attachments/assets/63950bd2-61c7-4cc6-b15e-bb7710b3fa6b)

Reply:
This was removed, we currently don't support LDAP/SAML handling of custom attributes.